openassessment/xblock/submission_mixin.py

```diff
@@ -603,6 +605,16 @@ def submission_path_and_context(self):
             context['save_status'] = self.save_status
             context['enable_delete_files'] = True
 
+            if self.teams_enabled:
+                try:
+                    team_info = self.get_team_info()
+                    if team_info:
+                        context.update(team_info)
+                except ObjectDoesNotExist:
+                    logger.error('User associated with anonymous_user_id can not be found')
```

Comment:
Nit: it'd be helpful to put the anonymous_user_id and/or `self.location` in this error message.